WordPress/src/main/java/org/wordpress/android/ui/mediapicker/MediaPickerViewModel.kt

```diff
@@ -43,10 +43,12 @@ import org.wordpress.android.ui.mediapicker.MediaPickerSetup.DataSource.STOCK_LI
 import org.wordpress.android.ui.mediapicker.MediaPickerSetup.DataSource.WP_LIBRARY
 import org.wordpress.android.ui.mediapicker.MediaPickerUiItem.ClickAction
 import org.wordpress.android.ui.mediapicker.MediaPickerUiItem.FileItem
+import org.wordpress.android.ui.mediapicker.MediaPickerUiItem.NextPageLoader
 import org.wordpress.android.ui.mediapicker.MediaPickerUiItem.PhotoItem
 import org.wordpress.android.ui.mediapicker.MediaPickerUiItem.ToggleAction
 import org.wordpress.android.ui.mediapicker.MediaPickerUiItem.VideoItem
 import org.wordpress.android.ui.mediapicker.MediaPickerViewModel.BrowseMenuUiModel.BrowseAction
+import org.wordpress.android.ui.mediapicker.MediaPickerViewModel.PhotoListUiModel.Data
```

Comment:
Minor (🔍): Consider removing this import and reverting to the existing approach of using `PhotoListUiModel.Data` instead of just `Data` as it saves us an import and having `PhotoListUiModel.Data` makes it more explicit, in terms on knowing what `Data` is that.

Reply:
great observation 🙌🏾, making changes now

Reply:
Awesome, thank you for applying my suggestion! 🙇 

PS: Next time don't forget to mention the commit that fixes this comment as it will make it easier for the reviewer to resolve the converstation, like that: 2971ffcd44079cf47a1bb58c8a0dcf4050c11827